.gitignore

```diff
@@ -67,6 +67,7 @@ ipch
 
 # Clion / IntelliJ project files
 /.idea/
+cmake-build-debug/
```

Comment:
Out of curiosity: when is this generated?

Reply:
CLion gernerated this folder by default. Pretty handy if can add it to `.gitigore`. I have been keep it locally for a long time.